client/src/components/QuestionsAndAnswers/QuestionListItem/QuestionListItem.jsx

```diff
@@ -3,14 +3,56 @@ import './QuestionListItem.scss';
 import QuestionListItemAnswer from '../QuestionListItemAnswer/QuestionListItemAnswer.jsx';
```

Comment:
I see your scss file but it is not imported into the React component. Is it intentional? If yes, ignore this comment :)

Reply:
The scss files are imported to the components. I see that we are doing it on slightly different ways but the way I'm doing also works.